services/blockstorage/test/block_sync_server_test.go

```diff
@@ -22,7 +22,12 @@ import (
 func TestSourceRespondToAvailabilityRequests(t *testing.T) {
 	test.WithContext(func(ctx context.Context) {
 		sourceAddress := keys.EcdsaSecp256K1KeyPairForTests(4).NodeAddress()
-		harness := newBlockStorageHarness(t).withNodeAddress(sourceAddress).withSyncBroadcast(1).start(ctx)
+		harness := newBlockStorageHarness(t).
+			withNodeAddress(sourceAddress).
+			withSyncBroadcast(1).
+			withValidateConsensusAlgosAtLeast(0).
```

Comment:
maybe I'm missing something but are you ever passing a non-zero values?
anyway, I'd create a special modifier for this use case, something like `ignoringHandleBlockConsensus()`

Reply:
renamed to `expectValidateConsensusAlgos()`